domain/lib/src/extension/string_extension.dart

```diff
@@ -33,4 +33,10 @@ import 'package:domain/domain.dart';
 extension StringExtension on String {
   int compareToSort(String value, OrderType orderType) =>
       toLowerCase().compareTo(value.toLowerCase()) * (orderType == OrderType.ascending ? -1 : 1);
+
+  String displayFileName()
```

Comment:
- Need to reformat code to have better looking. 
- Check lint for coding convention. 
- And also set meaningful name for the function

For e.g:
```
  String formatFileNameEllipsis() {
    final subLength = length - split('.').last.length;
    return '...' + substring(subLength - 3, subLength) + split('.').last;
  }
```

Reply:
`toCenterEllipsis()`